main/src/main/resources/report.html

```diff
@@ -0,0 +1,229 @@
+
+        .info:before {
+            content: "\26AA  ";
+        }
+
+        * {
+            box-sizing: border-box;
+        }
+
+        body {
+            padding: 1em 2em;
+        }
+
+        table {
+            width: 100%;
+        }
+        table th {
+            text-align: left;
+            border-bottom: 2px solid #000;
+            padding: 0.5em;
+        }
+        table td {
+            border-bottom: 1px solid #ddd;
+            padding: 0.5em;
+        }
+
+        table.accordion > tbody > tr.notice td, table.accordion > tbody > tr.view th {
+            cursor: auto;
+        }
+        table.accordion > tbody > tr.notice td:first-child,
+        table.accordion > tbody > tr.notice th:first-child {
+            position: relative;
+            padding-left: 20px;
+        }
+        table.accordion > tbody > tr.notice td:first-child:before,
+        table.accordion > tbody > tr.notice th:first-child:before {
+            position: absolute;
+            top: 50%;
+            left: 5px;
+            width: 9px;
+            height: 16px;
+            margin-top: -8px;
+            color: #000;
+            content: "+";
+        }
+        table.accordion > tbody > tr.notice.open td:first-child:before,
+        table.accordion > tbody > tr.notice.open th:first-child:before {
+            content: "\2013";
+        }
+        table.accordion > tbody > tr.notice:hover {
+            background: #ddd;
+        }
+        table.accordion > tbody > tr.notice.open {
+            background: #ddd;
+            color: black;
+        }
+        table.accordion > tbody > tr.description {
+            display: none;
+        }
+        table.accordion > tbody > tr.description.open {
+            display: table-row;
+        }
+
+        .desc-content {
+            padding: 0.5em;
+            border-bottom: 5px solid #000;
+            border-top: 5px solid #000
+        }
+        .desc-content h3 {
+            margin-top: 0;
+        }
+    </style>
+</head>
+
+<body>
+<h1>VALIDATION REPORT</h1>
+<h2><span th:text="${summary.noticeCount}">X</span> notices reported
+    (<span th:text="${summary.errorCount}">0</span> errors,
+    <span th:text="${summary.warningCount}">0</span> warnings,
+    <span th:text="${summary.infoCount}">0</span> infos)</h2>
+<p>Please visit <a href="https://github.com/MobilityData/gtfs-validator/blob/master/docs/NOTICES.md">NOTICES.md</a> and <a href="https://github.com/MobilityData/gtfs-validator/blob/master/RULES.md">RULES.md</a>.</p>
+<table class="accordion">
+    <thead>
+    <tr>
+    <th>Notice Code</th><th>Notice Name</th><th>Severity</th><th>Total</th>
+    </tr>
+    </thead>
+    <tbody>
+    <span th:each="noticesBySeverityLevel: ${summary.noticesMap}">
+        <span th:each="noticeByCode: ${noticesBySeverityLevel.value}">
+            <tr class="notice">
+                <td th:text="${noticeByCode.key}" />
+                <td th:text="${noticeByCode.value.get(0).getName()}" />
```

Comment:
Both here and throughout the HTML file, I think it is worth being consistent with the use of method calls vs OGNL property/getter syntax.  I think in OGNL style, you'd just have "noticesByCode.value[0].name" I believe?

Obviously, if you are actually calling a method (ala toLowerCase() or something similar), you'd leave it as a method call.

Reply:
Thanks for this. I fixed it